core/src/test/java/smile/data/BostonHousing.java

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
Please fix the header. Similarly for other files. Thanks

Reply:
I do not add particular code to handle headers manually. just use `Read.arff` as you suggest. Do I miss anything?

Reply:
The license header of source code file is ill-formated.